src/components/App/App.component.jsx

```diff
@@ -9,26 +10,26 @@ import SecretPage from '../../pages/Secret';
 import Private from '../Private';
 import Fortune from '../Fortune';
 import Layout from '../Layout';
-import { random } from '../../utils/fns';
+// import { random } from '../../utils/fns';
 
 function App() {
-  useLayoutEffect(() => {
-    const { body } = document;
+  // useLayoutEffect(() => {
+  //   const { body } = document;
 
-    function rotateBackground() {
-      const xPercent = random(100);
-      const yPercent = random(100);
-      body.style.setProperty('--bg-position', `${xPercent}% ${yPercent}%`);
-    }
+  //   function rotateBackground() {
+  //     const xPercent = random(100);
+  //     const yPercent = random(100);
+  //     body.style.setProperty('--bg-position', `${xPercent}% ${yPercent}%`);
+  //   }
 
-    const intervalId = setInterval(rotateBackground, 3000);
-    body.addEventListener('click', rotateBackground);
+  //   const intervalId = setInterval(rotateBackground, 3000);
+  //   body.addEventListener('click', rotateBackground);
 
-    return () => {
-      clearInterval(intervalId);
-      body.removeEventListener('click', rotateBackground);
-    };
-  }, []);
+  //   return () => {
+  //     clearInterval(intervalId);
+  //     body.removeEventListener('click', rotateBackground);
+  //   };
+  // }, []);
```

Comment:
Don't be afraid of deleting the code instead of commenting it 😅 you're probably are not going to need it, an if that's the case you can always go back to this PR as a reference.